frontend/src/Pages/ProgramOverviewDashboard.tsx

```diff
@@ -330,104 +326,126 @@ export default function ProgramOverviewDashboard() {
 
             {/* classes table */}
             <div className="card p-4">
-                <table className="table w-full table-fixed  mb-4">
-                    <thead className="bg-background">
-                        <tr>
-                            <th className="w-[150px]">
+                <table
+                    className="table-2 mb-4"
+                    style={{
+                        display: 'table',
+                        tableLayout: 'fixed',
+                        width: '100%'
+                    }}
+                >
+                    {/* colgroup allows us to manually set column widths of table-2 as long as style of table is set to display: table*/}
+                    <colgroup>
+                        <col style={{ width: '2%' }} /> {/* Checkbox */}
```

Comment:
@corypride This might be causing some issues for the click area. Clicking just outside of this navigates the user to the class overview page.